src/pages/tokens/[id].tsx

```diff
@@ -0,0 +1,178 @@
+          <div className='break-all'>{token.destruction.tx}</div>
+          <CopyButton text={token.destruction.tx} />
+        </AdaptiveList.Cell>
+      </AdaptiveList.Row>
+    </AdaptiveList>
+  )
+}
+
+function ListLeft ({ token }: { token: TokenData }): JSX.Element {
+  return (
+    <AdaptiveList>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Category'>{token.isDAT ? 'DAT' : 'LPS'}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Symbol'>{token.symbolKey}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Net Supply'>{token.limit}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Mintable'>
+          {(() => {
+            if (token.mintable) {
+              return (
+                <span className='flex flex-wrap items-center'>Yes <IoCheckmarkCircleOutline className='h-4 w-4 text-green-500 ml-1' /></span>
+              )
+            }
+            return (
+              <span className='flex flex-wrap items-center'>No <IoAlertCircleOutline className='h-4 w-4 text-gray-500 ml-1' /></span>
+            )
+          })()}
+        </AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Minted'>{token.minted}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Creation Height'>{token.creation.height}</AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      <AdaptiveList.Row>
+        <AdaptiveList.Cell name='Creation Tx' className='flex space-x-3 items-center'>
+          <div className='break-all'>{token.creation.tx}</div>
+          <CopyButton text={token.creation.tx} />
+        </AdaptiveList.Cell>
+      </AdaptiveList.Row>
+      {(token.collateralAddress !== undefined) && (
+        <AdaptiveList.Row>
+          <AdaptiveList.Cell name='Collateral Address' className='flex space-x-3 items-center'>
+            <div className='break-all'>{token.collateralAddress}</div>
+            <CopyButton text={token.collateralAddress} />
+          </AdaptiveList.Cell>
+        </AdaptiveList.Row>
+      )}
+    </AdaptiveList>
+  )
+}
+
+function CopyButton ({ text }: { text: string }): JSX.Element {
+  return (
+    <button
+      onClick={async () => await navigator.clipboard.writeText(text)}
+      className='cursor-pointer outline-none p-1 bg-gray-100 rounded shadow-sm'
+    >
+      <IoCopyOutline className='h-5 w-5 text-gray-500' />
+    </button>
+  )
+}
+
+export async function getServerSideProps (context: GetServerSidePropsContext): Promise<GetServerSidePropsResult<TokenAssetPageProps>> {
+  const api = getWhaleApiClient(context)
+  const id = context.params?.id?.toString() as string
+  const token = await api.tokens.get(id)
+  const poolPair = await api.poolpairs.list()
```

Comment:
not a good idea, 
- not a scalable design, what if we add more pool pair?
- not a performant design, sending necessary data

Let's remove this until we fixed the above